doc/playbook.rst

```diff
@@ -148,6 +149,46 @@ At the HEAD of the ``main`` branch, create and push a tag with the semantic vers
    git tag -s X.Y.Z -m "X.Y.Z"
    git push --tags
 
+Patch Releases and Release Branches
+-----------------------------------
+
+During commissioning and operations, it may be necessary to quickly deploy a bug fix without making any other changes that might potentially introduce new breakages.
+This can be done using a patch version (``X.Y.Z``).
+If there have been unrelated changes committed since the last release, you will need to isolate the bug fixes on a release branch.
+
+If the repo does not already have a release branch, create one anchored at the corresponding minor version tag:
+
+.. code-block:: sh
+
+   git checkout -b releases/X.Y X.Y.0
+   git push -u origin releases/X.Y
+
+If you have a branch for your bug fix, you can make a copy for the release branch:
+
+.. code-block:: sh
+
+   git checkout -b tickets/DM-XXXXX-X.Y tickets/DM-XXXXX
```

Comment:
Here and later, did you mean:
`git checkout -b tickets/DM-XXXXX-X.Y.Z tickets/DM-XXXXX`

Reply:
No, that was intentional -- it's the copy of `tickets/DM-XXXXX` being merged into the `X.Y` branch (by analogy to the `-v23` suffix in the Science Pipelines policy).

Reply:
Also, GitHub tracks each reviewer's approval/disapproval separately. 🙂